### PR TITLE
New Logo

### PR DESCRIPTION
**Pull request by Tobaloidee**
Thanks so much!

### CODE REVIEW

**Comment by luizdepra** [comment]
Can you plase change the image names to use only lower case characters and `-`? It is kind of a 'good pattern' for the web.

`HUGO CODER FAVICON.svg` -> `hugo-coder-favicon.svg`

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![hugo-coder logotype a](https://user-images.githubusercontent.com/36028424/40582328-583f656c-61a5-11e8-8d86-6e7c23f2b45d.png)
```
Comment (luizdepra) [inline README.md]: Please, fix this URL to something like this... following the screenshot example.
`https://github.com/luizdepra/hugo-coder/blob/master/images/hugo-coder-logotype-a.png`

 Also, you can remove the Hugo Coder title of the next line. Only showing the logo is enough.

**Comment by luizdepra** [comment]
I agree, @khos2ow.

**Comment by Tobaloidee** [comment]
I changed the location of the files to static/images/logo and renamed all the files to lowercases..
I had a hard time figuring out how to put the proper URL for the readme. I changed it back to the original layout.

**Comment by luizdepra** [comment]
OK. No problem. I can change the README.md later.

Thank you very much.

**Comment by khos2ow** [comment]
Another note, it feels to me `hugo-coder-` is redundant in the name of the files.

File: static/images/logo/logo [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (khos2ow) [inline static/images/logo/logo]: What's the purpose of this empty file? Is this a mistake commit?

**Pull request merged**